packages/kit/src/runtime/client/router.js

```diff
@@ -109,12 +109,16 @@ export class Router {
 		const handle_mousemove = (event) => {
 			clearTimeout(mousemove_timeout);
 			mousemove_timeout = setTimeout(() => {
-				trigger_prefetch(event);
+				// ensure event.composedPath() in find_anchor(event) returns an array of EventTarget objects
```

Comment:
I didn't understand the comment until I read through the issue thread
```suggestion
				// event.composedPath(), which is used in find_anchor, will be empty if the event is triggered in a timeout
				// add a layer of indirection to address that
```